cybersecuritytools/aws_permissions_helper/aws_requests.py

```diff
@@ -0,0 +1,9 @@
+import re
+import sys
+
+pattern = "((?<=DEBUG: Request\s)(\w*\W\w*))"
+
+with open(sys.argv[1],"r") as data:
+    with open('aws_requests.txt','w') as aws:
+        for s in sorted(set([re.search(pattern,d).group(1) for d in data if re.search(pattern, d)])):
```

Comment:
m8

Reply:
This is very clever, a bit too clever for me to understand. Would you rename or give names to the following variables:
```
pattern
sys.argv[1]
data
aws
s
d
``` 

The problem here is nothing is named, or the names add very little value.